components/gfx/paint_thread.rs

```diff
@@ -16,11 +16,10 @@ use euclid::size::Size2D;
 use font_cache_thread::FontCacheThread;
 use font_context::FontContext;
 use gfx_traits::color;
-use gfx_traits::{Epoch, LayerId, LayerKind, LayerProperties, PaintListener, PaintMsg as ConstellationMsg, ScrollPolicy};
+use gfx_traits::{Epoch, FrameTreeId, LayerId, LayerKind, LayerProperties, PaintListener, PaintMsg as ConstellationMsg, ScrollPolicy};
```

Comment:
This line here fails tidy. Lines cannot be over 120 characters. Use `./mach test-tidy` to check whether your code conforms to tidy rules.